pkg/reconciler/dependencybuild/dependencybuild.go

```diff
@@ -1054,6 +1058,27 @@ func (r *ReconcileDependencyBuild) createRebuiltArtifacts(ctx context.Context, l
 	return nil
 }
 
+func (r *ReconcileDependencyBuild) createArtifacts(ctx context.Context, log logr.Logger, pr *tektonpipeline.PipelineRun, db *v1alpha1.DependencyBuild, deployed []string) error {
+	db.Status.DeployedArtifacts = deployed
+
+	for _, i := range deployed {
+		ab := v1alpha1.ArtifactBuild{}
+		ab.Namespace = pr.Namespace
+		ab.Name = artifactbuild.CreateABRName(i)
+		ab.Annotations = map[string]string{artifactbuild.DependencyAnnotation: db.Name}
+		ab.Spec.GAV = i
+		if err := controllerutil.SetOwnerReference(&ab, db, r.scheme); err != nil {
```

Comment:
After further testing this isn't correct I think. It needs to be `controllerutil.SetOwnerReference(db, &ab, r.scheme)` but I don't think we can have it in this function as the caller calls patchStatus.

Reply:
I am trying an experimental change to update it elsewhere but currently hitting a mixture of rbac problems and 'object has been modified' ....